internal/ignore/path_matcher.go

```diff
@@ -54,6 +55,10 @@ func CreateFileChangeFilter(m IgnorableTarget) (model.PathMatcher, error) {
 	for _, r := range m.LocalRepos() {
 		matchers = append(matchers, git.NewRepoIgnoreTester(context.Background(), r.LocalPath))
 	}
+	// Also filter out any git directories in any dependencies
+	for _, p := range m.Dependencies() {
+		matchers = append(matchers, git.NewRepoIgnoreTester(context.Background(), p))
```

Comment:
out of curiosity, why does this take a context?

Reply:
No idea, it doesn't need to. Can change in a subsequent PR.